conda_forge_tick/migrators/version.py

```diff
@@ -315,19 +312,17 @@ def migrate(
                 )
             )
 
-    def pr_body(
-        self, feedstock_ctx: ClonedFeedstockContext, add_label_text: bool = False
-    ) -> str:
-        if feedstock_ctx.feedstock_name in self.effective_graph.nodes:
+    def pr_body(self, feedstock_ctx: ClonedFeedstockContext) -> str:
```

Comment:
We cannot remove the add_label_text keyword. We should instead use it in the `super` call at the end of the method.

Reply:
Resolved!